src/index.ts

```diff
@@ -1,20 +1,41 @@
-export { Pinecone, type PineconeConfiguration } from './pinecone';
+export { Pinecone } from './pinecone';
+export { Index } from './data';
 
 export * as Errors from './errors';
 
 // Type exports
-export type { CreateIndexOptions, IndexList, IndexName } from './control';
 export type {
+  ConfigureIndexOptions,
```

Comment:
**suggestion:** Maybe alphabetize these really chunky export lists? I'm not sure how far we want to go with alphabetizing, but I feel like it can be really helpful for bigger lists of attributes like this where we don't want to miss anything.